packages/x-components/src/x-modules/tagging/service/external-tagging.service.ts

```diff
@@ -3,25 +3,32 @@ import { Result } from '@empathyco/x-types';
 import { BrowserStorageService, StorageService } from '@empathyco/x-storage-service';
 import { RootXStoreState } from '../../../store/index';
 import { XPlugin } from '../../../plugins/index';
-import { PDPAddToCartService } from './types';
+import { ExternalTaggingService } from './types';
 
 /**
- * Default implementation for the {@link PDPAddToCartService}.
+ * Default implementation for the {@link ExternalTaggingService}.
  *
  * @public
  */
-export class DefaultPDPAddToCartService implements PDPAddToCartService {
+export class DefaultExternalTaggingService implements ExternalTaggingService {
   /**
-   * Session id key to use as key in the storage.
+   * Session id key to use as key in the storage for result clicks.
    *
    * @public
    */
   public static readonly RESULT_CLICKED_ID_KEY = 'add-to-cart';
 
   /**
-   * Global instance of the {@link PDPAddToCartService}.
+   * Session id key to use as key in the storage for add to carts.
```

Comment:
Here specify that means clicking add2cart button:
```suggestion
   * Session id key to use as key in the storage for the add to cart button's clicks.
```

Reply:
This is fine, IMO, because it is not only for the add to cart button but also for programmatically add to carts (`InterfaceX.addProductToCart()`)